### PR TITLE
add option to force destroy bucket even is not empty

### PR DESCRIPTION
**Pull request by rafilkmp3**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -57,3 +57,8 @@ variable "tags" {
   description = "A map of tags (in addition to Name) to add to all resources."
   default     = {}
 }
+
+variable "force_destroy" {
+  description = "force destroy bucket even bucket its not empty"
```
Comment (Cayan) [inline variables.tf]: force to destroy the bucket even if it is not empty

**Review by Cayan — CHANGES_REQUESTED** [review]
review the description for the variable

File: cloudfront.tf [hunk 2]
```diff
@@ -62,6 +62,22 @@ resource "aws_cloudfront_distribution" "website" {
     prefix          = "${var.hostname}/cloudfront"
   }
 
+  price_class = "PriceClass_100"
```
Comment (jonathanio) [inline cloudfront.tf]: Can you remove the additional `price_class` here as it overrides the version available above, and in turn hard-codes support to `PriceClass_100` when the above version of configurable.

File: cloudfront.tf [hunk 3]
```diff
@@ -62,6 +62,22 @@ resource "aws_cloudfront_distribution" "website" {
...
+
+  custom_error_response {
```
Comment (jonathanio) [inline cloudfront.tf]: Can you update the `custom_error_response` lines here too. I'm uncomfortable with them being unconfigurable as firstly they also don't point to `${var.error_document}` (which is used in the S3 configuration for the content bucket) and you should not change the response code - `404` and `400` are valid responses; browsers and client devices should be sent them to deal with appropriately.

Comment (rafilkmp3) [reply cloudfront.tf]: sorry about this, my first commit was just to fix what I propose in pull request but i host a angular application and I will move to another branch. what I propose is only the force destroy directive.

Comment (jonathanio) [reply cloudfront.tf]: That's OK. You'll need to either delete the last few commits on this branch and force push it back up to remove them from the pull request or revert them and I'll squash them on the merge.

**Comment by rafilkmp3** [comment]
Hi could check if we can merge now ?

File: cloudfront.tf (hunk 3 above)
Comment (rafilkmp3) [reply cloudfront.tf]: done sorry about the late

**Comment by jonathanio** [comment]
@rafilkmp3,

Sorry, I hadn't seen the update come through. I'll merge and release now.

**Pull request merged**